### PR TITLE
Option to show only `user` qubes in Qube Manager

### PR DESCRIPTION
**Pull request by alimirjamali**
fixes: https://github.com/QubesOS/qubes-issues/issues/8663

### CODE REVIEW

**Comment by alimirjamali** [comment]
Gitlab CI is complaining on Pylint. When I try to open the test, it shows me error 500 (`the server encountered an unexpected condition that prevented it from fulfilling the request.`). Running pylint locally does not show any errors. I do not have access to retrypipeline failed. Advise is needed.

**Comment by alimirjamali** [comment]
PipelineRetry

File: ui/qubemanager.ui [hunk 1]
```diff
@@ -317,6 +317,19 @@ template</string>
...
+      <item>
+       <widget class="QCheckBox" name="show_user">
+        <property name="text">
+         <string>User qubes</string>
+        </property>
+        <property name="checked">
+         <bool>true</bool>
+        </property>
+        <property name="toolTip">
+         <string>+AppVMs +Standalones&lt;br&gt;-Disposables -ServiceVMs</string>
```
Comment (marmarta) [inline ui/qubemanager.ui:329]: Maybe "Show AppVMs and Standalone VMs, hide disposable and service VMs"?

**Comment by marmarta** [comment]
The weird 500 issue was, I think, some CI hiccup, should be fine now. Also: sorry for the long delay on review on this and other PRs, I've had shitty health stuff happen and only now I'm slowly digging myself from the sick leave hole :)

File: ui/qubemanager.ui (hunk 1 above)
Comment (alimirjamali) [reply ui/qubemanager.ui:329]: > Maybe "Show AppVMs and Standalone VMs, hide disposable and service VMs"?

Yes. This is much much better. I did not like the old message personally.

**Comment by marmarta** [comment]
Pylint complains, but that is fixed in https://github.com/QubesOS/qubes-manager/pull/387 (and apparently https://github.com/QubesOS/qubes-manager/pull/390 )

**Review by marmarta — APPROVED** [review]

**Comment by marmarek** [comment]
PipelineRetry

**Pull request merged**